frontend/src/modules/Worksheets/views/WorksheetView.js

```diff
@@ -181,11 +181,19 @@ const WorksheetView = () => {
           response.data.searchEnvironmentDataItems.nodes.map((d) => ({
             datasetUri: d.datasetUri,
             value: d.datasetUri,
-            label: `${d.GlueDatabaseName}_shared_${d.shareUri}`,
-            GlueDatabaseName:
-              `${d.GlueDatabaseName}_shared_${d.shareUri}`.substring(0, 254),
+            label: `${d.GlueDatabaseName}_shared`,
+            GlueDatabaseName: `${d.GlueDatabaseName}_shared`,
```

Comment:
So this will always show `${d.GlueDatabaseName}_shared` now - even if it is an old share correct?

Reply:
oops, completely blocked that from my mind. Fixing it

Reply:
Updated (see description of PR)